src/wp-includes/block-template.php

```diff
@@ -17,6 +17,41 @@ function _add_template_loader_filters() {
 	}
 }
 
+/**
+ * Renders a warning screen for empty block templates.
+ *
+ * @since x.x
+ *
+ * @param WP_Block_Template $block_template The block template object.
+ * @return string The warning screen HTML.
+ */
+function wp_render_empty_block_template_warning( $block_template ) {
+	wp_enqueue_style( 'wp-empty-template-alert' );
+	return sprintf(
+		/* translators: %1$s: Block template title. %2$s: Empty template warning message. %3$s: Edit post link. %4$s: Edit button label. */
+		'<div id="wp-empty-template-alert">
+			<div class="icon">
+				<span class="dashicons dashicons-warning"></span>
+			</div>
+			<div class="content">
+				<div class="message">
+					<h2>%1$s</h2>
+					<p>%2$s</p>
+				</div>
+				<div class="actions">
+					<a href="%3$s" class="wp-element-button">
+						%4$s
+					</a>
+				</div>
+			</div>
+		</div>',
+		$block_template->title,
+		__( 'This page is blank because your theme\'s template is currently empty. It may have been accidentally deleted, but don\'t worry! You can easily reset or edit this template in the <a href="https://wordpress.org/documentation/article/site-editor/">site editor</a>.' ),
```

Comment:
We can simplify this. I don't think we should infer what the user may or may not have done, but give them the information to understand. Also "Site Editor" is not a link. The button serves the purpose of bringing them back to the proper experience. 

```
This page is blank because the theme’s template is empty. You can reset or customize it in the Site Editor.
```

Reply:
I agree with keeping this wording simple 👍 This sentence sounds good to me. These changes are rolled into [this commit](https://github.com/WordPress/wordpress-develop/pull/7354/commits/63ddc94188ec5d5b98467d8e3215ace55e3c4f80).